scripts/build-docs.sh

```diff
@@ -21,12 +21,10 @@ appledoc \
     --no-warn-invalid-crossref \
     --no-warn-undocumented-object \
     --no-warn-undocumented-member \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
-    --ignore "Realm/RLMArrayAccessor.h" \
-    --ignore "Realm/RLMArrayAccessor.mm" \
-    --ignore "Realm/RLMQueryUtil.h" \
-    --ignore "Realm/RLMUtil.h" \
-    --ignore "Realm/RLMRealm_Dynamic.h" \
+    --ignore ".m" \
```

Comment:
> since they are now more public they should be in there but with sufficient warnings (came out of conversant with Tim).

Sure, but the methods get added inline with the public ones, in alphabetical order, so it's not exactly well delineated.